docs/tutorial/new-tutorial.md

```diff
@@ -0,0 +1,384 @@
+(tutorial)=
+# SQLair tutorial
+
+Imagine you are writing a Go program that handles your customers and their
+orders. The business logic is contained in the Go program but the customer and
+order data is stored in a SQL database. In this tutorial, we'll see how easy
+SQLair makes it to map the database between the database and the Go program.
+Let's get started!
+
+
+```{note}
+This tutorial will take around 30 minutes to complete. If you want a quicker
+start see {ref}`prepare-and-run`.
+```
```

Comment:
If that doc really acts as a quicker start, something is wrong there.

Reply:
That doc just shows you how to prepare and run a query, which may be all they want to do. I guess its not really a quick start, just a proof of concept kind of thing. I'll remove the link.